src/content/blog/2022-08-08-new-config-system-part-3.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "ESLint's new config system, Part 3: Developer preview"
+teaser: "While the new config system isn't yet incorporated into the CLI, it is available via API for developers to test."
+tags:
+  - Configuration
+  - Command Line
+authors:
+  - nzakas
+categories:
+  - API Changes
+---
+
+In my [previous post](https://eslint.org/blog/2022/08/new-config-system-part-2/), I explained the fundamental concepts of using the new "flat" config system. The new config system isn't yet tied into the API while do more internal testing, but we did want to give the ESLint community a chance to experiment with flat config while we work on incorporating it into the CLI. So ESLint v8.21.0 incorporates several ways to try out flat config as we work on it. Please keep in mind that everything mentioned in this post is experimental and we would love your feedback as you try it out.
```

Comment:
`The new config system isn't yet tied into the API while do more internal testing` I dont get this part? am I missing something?

Reply:
Oops, should he CLI, not API.